docs/affinityassistants.md

```diff
@@ -27,10 +28,10 @@ and only one PipelineRun is allowed to run on a node at a time.
 This means that Affinity Assistant is incompatible with other affinity rules
 configured for the `TaskRun` pods (i.e. other affinity rules specified in custom [PodTemplate](pipelineruns.md#specifying-a-pod-template) will be overwritten by Affinity Assistant).
 If the `PipelineRun` has a custom [PodTemplate](pipelineruns.md#specifying-a-pod-template) configured, the `NodeSelector` and `Tolerations` fields will also be set on the Affinity Assistant pod. The Affinity Assistant
-is deleted when the `PipelineRun` is completed. 
+is deleted when the `PipelineRun` is completed.
 
-Currently, the Affinity Assistant Modes can be configured by the `disable-affinity-assistant` and `coschedule` feature flags. 
-The `disable-affinity-assistant` feature flag is now deprecated and will be removed in release `v0.60`. At the time, the Affinity Assistant Modes will be only determined by the `coschedule` feature flag. 
+The Affinity Assistant Modes are configured by the `coschedule` feature flag.
+Previously, it was also controlled by the `disable-affinity-assistant` feature flag which was deprecated and removed after release `v0.68`.
```

Comment:
I believe this needs to go in a note as it's not directly useful while configuring the affinity assistant for user at the time of reading the docs for affinity assistant.

Reply:
Is it okay to write like this in the notes?

```
Action Required: The feature flag `disable-affinity-assistant` has been removed. Please use `coschedule` as a replacement.
It was marked as deprecated in version v0.51 and has been completely removed in the current version.
If you need to maintain the previous default behavior, set feature flag `coschedule` to `disabled`.
For specific behavioral differences, please refer to: https://github.com/tektoncd/community/blob/main/teps/0135-coscheduling-pipelinerun-pods.md#tep-0135-coscheduling-pipelinerun-pods
```

Reply:
that'll work 😸